lib/shared/bucketing-assembly-script/assembly/types/bucketedUserConfig.ts

```diff
@@ -106,6 +107,7 @@ export class SDKFeature extends JSON.Obj {
             getStringFromJSON(feature, 'type'),
             getStringFromJSON(feature, 'key'),
             getStringFromJSON(feature, '_variation'),
+            getStringFromJSONOptional(feature, 'variationName'),
```

Comment:
does the variationName need to be optional? Is it not a required field in the config data?

Reply:
ah true, it's optional in a few places in the types (like in @devcycle/types.../feature.ts) but it's required on the actual mongo schema so i think it's safe to assume it's actually required

Reply:
oh wait never mind - we have it optional here https://github.com/DevCycleHQ/js-sdks/blob/main/lib/shared/bucketing-assembly-script/assembly/types/feature.ts#L49 so i think it needs to be optional

Reply:
we can change that to required too. We know that variation name will always appear in the project config (due to the schema of the model), so we should change the config types to represent that

Reply:
Makes sense, fixed!